src/qat/purr/backends/live.py

```diff
@@ -82,8 +82,23 @@ def apply_setup_to_hardware(
         qubit = Qubit(primary_index, resonator, ch1)
         qubit.create_pulse_channel(ChannelType.drive, frequency=5.5e9)
 
+        grid_time = qubit.physical_channel.grid_time
```

Comment:
can you specify which time units we're dealing with here ?
usually it's `name_nanos` or `name_seconds`
integer arithmetic suggests the unit is nanos but just to be explicit